app/views/series/_series_order.html.erb

```diff
@@ -5,7 +5,7 @@
       <li id="serial_<%= serial.id %>" class="serial-position-list">
         <%= text_field_tag 'serial_works[]', nil, :size => 3, :maxlength => 3, :class => 'number serial-position-field', :id => "serial_#{i}" %>
         <span id='position-for-<%= serial.id %>'><%= serial.position %></span>.
-        <h3 class="heading"><%=serial.work.title.html_safe + (serial.work.posted? ? "" : " (DRAFT)") %></h3>
+        <h3 class="heading"><%= serial.work.title + (serial.work.posted? ? "" : " (DRAFT)") %></h3>
```

Comment:
Since you're here (and this page isn't cached), could you update this to use the I18n `t` helper as well? One idea I had (it may not be the best way to do things though)
```suggestion
        <h3 class="heading"><%= serial.work.posted? ? serial.work.title : t(".draft_work_title", title: serial.work.title)) %></h3>
```

Another way I thought of, if we don't mind having a "useless" translation key:
```suggestion
        <h3 class="heading"><%= t(serial.work.posted? ? ".published_work_title" : ".draft_work_title", title: serial.work.title)) %></h3>
```
(where the `"published_work_title" key is just `published_work_title: '%{title}'`)